tutorials/algorithms/01_algorithms_introduction.ipynb

```diff
@@ -116,19 +117,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from qiskit.utils import QuantumInstance\n",
-    "\n",
-    "backend = Aer.get_backend('aer_simulator')\n",
-    "quantum_instance = QuantumInstance(backend=backend, shots=800, seed_simulator=99) "
+    "vqe = VQE(estimator, ansatz, optimizer)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note: if you provide the backend directly then internally a `QuantumInstance` will be created from it, with default settings, so at all times the algorithms are working through a `QuantumInstance`.\n",
-    "\n",
-    "So now we would be able to call the [compute_mininum_eigenvalue()](https://qiskit.org/documentation/stubs/qiskit.algorithms.VQE.html#qiskit.algorithms.VQE.compute_minimum_eigenvalue) method. The latter is the interface of choice for the application modules, such as Nature and Optimization, in order that they can work interchangeably with any algorithm within the specific category."
+    "Now we would be able to call the [compute_mininum_eigenvalue()](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQE.compute_minimum_eigenvalue.html#qiskit.algorithms.minimum_eigensolvers.VQE.compute_minimum_eigenvalue) method. The latter is the interface of choice for the application modules, such as Nature and Optimization, in order that they can work interchangeably with any algorithm within the specific category."
```

Comment:
```suggestion
    "Now we should be able to call the [compute_mininum_eigenvalue()](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQE.compute_minimum_eigenvalue.html#qiskit.algorithms.minimum_eigensolvers.VQE.compute_minimum_eigenvalue) method. The latter is the interface of choice for the application modules, such as Nature and Optimization, in order that they can work interchangeably with any algorithm within the specific category."
```

Reply:
Reworded to `Now we can call ...` in [09051a4](https://github.com/Qiskit/qiskit-tutorials/pull/1370/commits/09051a41886769c541bbd227c62764cc63925ea4)